pandas-stubs/core/indexes/base.pyi

```diff
@@ -313,7 +312,13 @@ class Index(IndexOpsMixin[S1]):
     def rename(self, name, inplace: Literal[True]) -> None: ...
     @property
     def nlevels(self) -> int: ...
-    def sortlevel(self, level=..., ascending: bool = ..., sort_remaining=...): ...
+    def sortlevel(
+        self,
+        level=...,
+        ascending: bool = ...,
+        sort_remaining=...,
+        na_position: Literal["first", "last"] = ...,
+    ): ...
```

Comment:
I think we should delete this from `Index`, because `sortlevel()` is only documented for `MultiIndex` and only makes sense in that context

Reply:
I was considering that but thought it would make too many changes in one PR, agree with your call let me adjust!